Zend/zend_types.h

```diff
@@ -533,6 +533,7 @@ typedef uint32_t HashPosition;
 typedef struct _HashTableIterator {
 	HashTable    *ht;
 	HashPosition  pos;
+	uint32_t      next_copy;
```

Comment:
Please add a comment, that copies are linked into a circular list. (If I understood this properly).